rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -180,7 +181,10 @@ namespace rerun {
 
             std::vector<DataCell> data_cells;
             data_cells.reserve(sizeof...(Ts));
-            push_data_cells(data_cells, component_array...);
+            const auto status = push_data_cells(data_cells, component_array...);
+            if (!status) {
```

Comment:
I vote we should be more explicit

```suggestion
            if (status.is_err()) {
```